dashboard-frontend/src/helpers/api.ts

```diff
@@ -96,6 +94,32 @@ const createStudy = ({
     body: { name },
   });
 
+const createStudyConf = ({
+  data,
+  slug,
+  accessToken,
+}: {
+  data: StudyConfResource;
+  slug: string;
+  accessToken: string;
+}) =>
+  apiRequest<CreateStudyConfApiResponse>(`/studies/${slug}/conf`, {
+    accessToken,
+    method: 'POST',
+    body: { ...data },
```

Comment:
`body: data`, no?

Reply:
No that will make it:
```json
{
  "data": {
    "genera: {
    }
  }
}
```
When what we want is:
```json
{
  "general": {
    ...
  }
}
```

So I think this is correct

Reply:
Actually, this is turning out to be tricky! I tried spreading the data obj at the component level before passing it to the api call and I wasn't able to get this to return a response without using the spread operator on the `body`